packages/legend-graph/src/graph/MetaModelConst.ts

```diff
@@ -57,6 +57,19 @@ export enum PRIMITIVE_TYPE {
   LATESTDATE = 'LatestDate',
 }
 
+export enum ATOMIC_TEST_TYPE {
+  SERVICE_TEST = 'serviceTest',
+  MAPPING_TEST = 'mappingTest',
+}
+
+export enum TYPICAL_MULTIPLICITY_TYPE {
```

Comment:
why did we add this?

Reply:
because of cyclic dependency which was being caused. so you suggested in our previous conversation to use an enum directly mentioned at metamodel_const file, helps deserialization of data